playbooks/collect_instance_statistics/user_stats.py

```diff
@@ -0,0 +1,47 @@
+#!/edx/bin/python.edxapp
```

Comment:
@pcockwell, why should this script be duplicated? Can't the existing script be used as-is? Even if we don't want the `hits` field, we can either remove it from the output or rename it and add the elasticsearch hits field with a unique name. What do you think?

Reply:
Yes, the existing script can likely be used as-is by renaming the elasticsearch result fields